### DISCUSSION TITLE
unable to use react-flow

### DISCUSSION THREAD

**Discussion by ghost** [xyflow/xyflow#2805] [Q&A] [answered]
I have added react-flow as dependency into my project, here is the package.json
<img width="311" alt="image" src="https://user-images.githubusercontent.com/6542274/216751873-68c04b63-7bc6-43f1-9a99-2cd95867a66b.png">

post install, I tried to start my local as usual but I'm getting loaders issue from webpack. Do you know how I can resolve it?
<img width="1028" alt="Screenshot 2023-02-04 at 11 32 01 AM" src="https://user-images.githubusercontent.com/6542274/216751841-ae607960-6628-4d6e-8413-994cece8baeb.png">

**Comment by moklick** [comment, marked as answer]
We are not transpiling React Flow. You need to add it to your babel loader https://github.com/wbkd/react-flow/discussions/2795#discussioncomment-4855163

**Reply by ghost** [reply]
Thank you. Worked.
